examples/zstd-example-params.json

```diff
@@ -0,0 +1,8 @@
+{
+    "module": "affine-fusion",
```

Comment:
This example is incomplete without input data, so I'm not sure what it's useful for? It would be better to have a test profile that downloads test data (like nf-core pipelines).

Reply:
I will look into that and maybe add a test for it.